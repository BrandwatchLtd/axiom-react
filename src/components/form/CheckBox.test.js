import React from 'react';
import renderer from 'react-test-renderer';
import { CheckBox } from 'bw-axiom';

function getComponent(props = {}) {
  return renderer.create(
    <CheckBox { ...props }>Lorem ipsum</CheckBox>
  );
}

describe('CheckBox', () => {
  it('renders with defaultProps', () => {
    const component = getComponent();
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
