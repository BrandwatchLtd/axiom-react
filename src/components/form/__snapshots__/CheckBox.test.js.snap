// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CheckBox renders with defaultProps 1`] = `
<label
  className="ax-chedio-buttox__container ax-space--x2"
  onClick={undefined}
>
  <input
    className="ax-chedio-buttox ax-checkbox"
    disabled={undefined}
    type="checkbox"
  />
  <span
    className="ax-chedio-buttox__indicator ax-checkbox__indicator"
  />
  <span
    className="ax-chedio-buttox__label"
  >
    Lorem ipsum
  </span>
</label>
`;
